client/src/resources/style-constants.js

```diff
@@ -19,6 +19,10 @@ export const grey3 = "#e0e0e0"; // light gray
 export const link = "#2196f3"; // color for links
 export const highlightedLink = "#35baf6"; // color of links on hover
 export const errorRed = "#FF0002";
+export const eventOrange = "#FFA089";
+export const eventYellow = "#FFD958";
```

Comment:
Placing these colors here are fine for now, but I think eventually these colors should be stored on the backend and be pulled down as part of the "getEventInfo" response.

Reply:
Agreed, I just put it there for now so we have something to use.